galaxia_custom/data/request_link_data.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record id="req_link_hr_holidays" model="res.request.link">
+            <field name="name">Leaves</field>
+            <field name="object">hr.holidays</field>
+        </record>
+        <delete id="product.req_link_product" model="res.request.link"/>
```

Comment:
No hay error al actualizar, pero en el log muestra un "Skipping deletion for missing XML ID"